t/api/02-iso.t

```diff
@@ -968,24 +968,26 @@ subtest 'PUBLISH and STORE variables cannot include slashes' => sub {
 subtest 'Expand specified variables when scheduling iso' => sub {
     $schema->txn_begin;
     add_opensuse_test(
-        'test',
+        'foo',
         BUILD_HA            => '%BUILD%',
         BUILD_SDK           => '%BUILD_HA%',
         SHUTDOWN_NEEDS_AUTH => 1,
-        HDD_1   => '%DISTRI%-%VERSION%-%ARCH%-%BUILD_SDK%@%MACHINE%-minimal_with_sdk%BUILD_SDK%_installed.qcow2',
-        MACHINE => ['32bit', '64bit'],
+        HDD_1         => '%DISTRI%-%VERSION%-%ARCH%-%BUILD_SDK%@%MACHINE%-minimal_with_sdk%BUILD_SDK%_installed.qcow2',
+        MACHINE       => ['32bit', '64bit'],
+        YAML_SCHEDULE => '%TEST%@%MACHINE%-staging.yaml',
```

Comment:
Good call adding that. :+1: 

Maybe we should cover `TEST_SUITE_NAME` and `JOB_DESCRIPTION` here as well, since they are part of the same code path?

Reply:
Add the `TEST_SUITE_NAME` checking